proknow-sdk-test/LogTest/AuditTest.cs

```diff
@@ -99,5 +99,34 @@ public async Task QueryTest()
             Assert.AreEqual(nextPatientItem.WorkspaceName, $"{workspaceItem.Name}");
         }
 
+        [TestMethod]
+        public async Task FilterParametersTest()
+        {
+            FilterParameters filterParams = new FilterParameters();
+            filterParams.Types = new string[] { "patient_created" };
+            filterParams.PageSize = 1;
+            filterParams.StartTime = DateTime.Now;
+            filterParams.EndTime = DateTime.Now.AddDays(-1);
```

Comment:
Not sure it really matters for this test but `EndTime` should be later than `StartTime`